tms/home-landing-pages/admin-courses-and-activities-landing.md

```diff
@@ -90,6 +113,8 @@
       <a class="subtopic_link" href="/tms/tms-administrators/courses-and-activities/other-activities/start-scorm-module.md">
         <div class="subtopic_title">How can I preview a SCORM module belonging to my organization?</div>
       </a>
+      <a class="subtopic_link" href="/tms/tms-administrators/courses-and-activities/pgs/add-courses-to-publishing-group.md">
+        <div class="subtopic_title">How do I add a course to my publishing group?</div>
```

Comment:
</a> tag on 118 to close the <a class="subtopic_link">

Reply:
has been added